docs/journalist.rst

```diff
@@ -438,27 +437,33 @@ highly sensitive submissions.
 Removing Metadata
 ~~~~~~~~~~~~~~~~~
 
-.. tip:: For detailed information about removing metadata from documents, check out
-         this in-depth `guide to removing metadata`_.
+.. tip:: For detailed information about removing metadata from documents, check out this in-depth `guide to removing metadata`_.
 
-Tails comes with the `Metadata Anonymisation Toolkit`_ (MAT) that
+Tails comes with the `Metadata Anonymisation Toolkit 2`_ (MAT2) that
 is used to help strip metadata from a variety of types of files,
 including png, jpg, OpenOffice/LibreOffice documents, Microsoft Office
 documents, pdf, tar, tar.bz2, tar.gz, zip, mp3, mp2, mp1, mpa, ogg,
-and flac. You can open MAT by clicking **Applications** ▸ **System Tools** ▸
-**MAT**.
+and flac. We recommend using this and other tools to work with documents within Tails for as much of your workflow as possible.
 
-We recommend always doing as much work as possible inside of Tails
-before working with documents on your everyday workstation. This includes
-stripping metadata with MAT.
+Tails 4 replaces MAT with MAT2, which is usable via the command line and via a context menu in the **Files** application (called "nautilus" on the command line).
 
-.. warning:: MAT is no longer actively maintained and **will not**
-             strip all metadata, even when the output claims the
-             document is clean. Some metadata are likely to persist:
-             you must **never** assume MAT has removed all metadata.
+You can use MAT2 via the **Files** application by browsing to **Places** ▸ **(Your file's location)** and right-clicking on your file. In the context menu, select **Remove metadata**.
 
-When you no longer need documents, you can right-click on them and
-choose **Wipe** to delete them.
+|mat2 context menu|
+
+Note that this does not alter the original file--it creates a clean copy.
+
+|mat2 cleaned|
+
+To use MAT2 on the command line, type ``man mat2`` to see a list of available actions you can take with MAT2. For example, you can view the metadata of a file with ``mat2 myfile --show``.
+
+|mat2 cli show|
+
+You can create a "clean" version of the document with ``mat2 myfile``, again noting that this does not erase the metadata on the original file. This is equivalent to the "Remove metadata" context menu option.
+
+Note that even after running MAT2, you should carefully inspect files to ensure that all metadata has been wiped, or convert them to a simpler file format (for example, converting a ``.xls`` file to a ``.csv``) to ensure that metadata is not left behind in error.
+
+When you no longer need documents, in the Files application, you can right-click on them and choose **Wipe** to securely delete them.
```

Comment:
That middle clause is a bit ambiguous, how about "When you no longer need documents, right-click  them in the Files application and choose **Wipe** to securely delete them."